test/manual/test_credential_process_manual.clj

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+#_(
+   set -eu
+   
+   #_ YO Make sure you run this from the root of the repo!
+   
+   echo "Using AWS profile $AWS_PROFILE"
+
+   #_DEPS is same format as deps.edn. Multiline is okay.
+   DEPS='
+   {:deps {org.clojure/clojure         {:mvn/version "1.11.1"}
+           com.cognitect.aws/api       {:mvn/version "0.8.539"}
+           com.cognitect.aws/endpoints {:mvn/version "1.1.12.192"}
+           com.cognitect.aws/s3        {:mvn/version "822.2.1109.0"}
+           latacora/backsaws           {:local/root  "./"}}}
+   '
+
+   #_You can put other options here
+   OPTS='
+   -J-Xms256m -J-Xmx256m -J-client
+   '
+
+exec clojure $OPTS -Sdeps "$DEPS" -M "$0" "$@"
+)
```

Comment:
uhhh. Is this a shell script that pretends to be a clojure file (or, I suppose, the other way around)? that's a first.

Reply:
![why-not-both-animated-gif-7](https://user-images.githubusercontent.com/99206340/164082841-169acc20-d4ee-4943-a521-4e79dcb8f3fe.gif)

Reply:
Actually this technique was devised a couple of years ago by Eric Normand: https://gist.github.com/ericnormand/6bb4562c4bc578ef223182e3bb1e72c5